### DISCUSSION TITLE
Reproduce the given benchmark results

### DISCUSSION THREAD

**Discussion by yeliang2258** [vllm-project/vllm#500] [Q&A]
Hello，I would like to know which model and test tool are used for the performance data provided on the homepage? 
<img width="769" alt="image" src="https://github.com/vllm-project/vllm/assets/30516196/71c0b9a2-bb22-46d6-a7c9-2564528b68cc">

Is the model available on https://huggingface.co/?

**Comment by yeliang2258** [comment]
lmsys/vicuna-13b-v1.3, young-geng/koala, openlm-research/open_llama_13b, or others ?

**Comment by yeliang2258** [comment]
@zhuohan123 Please help me solve this problem, thank you

**Comment by zhuohan123** [comment]
Hi we use the official LLaMA model (e.g., `huggyllama/llama-13b`). However, all these llama models you listed should perform exactly the same.

**Comment by zhuohan123** [comment]
We use this script to get the benchmark results: https://github.com/vllm-project/vllm/blob/main/benchmarks/benchmark_serving.py

**Comment by yeliang2258** [comment]
https://github.com/vllm-project/vllm/blob/main/benchmarks/launch_tgi_server.sh
In addition, I would like to ask, in this script, what is the TOKENS setting?
